rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+wrap_comments = true
```

Comment:
Nice! Let's remove these config options from `tools/pre-push` and `.github/workflows/main.yml`.